lib/my_home_page.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:flutter/material.dart';
+import 'like_button.dart';
+import 'movie.dart';
+
+class MyHomePage extends StatelessWidget {
+  MyHomePage({super.key});
+
+  static const Color backgroundColor = Color.fromRGBO(
+    255,
+    255,
+    255,
+    0.8,
+  );
+  static const double appBarToolbarHeight = 50;
+  static const double posterWidth = 200;
+  static const double largePadding = 10;
+  static const double mediumPadding = 5;
+  static const double smallPadding = 2.5;
+  static const Icon appBarBackIcon = Icon(Icons.arrow_back);
+  static const double largeFont = 20;
+  static const double mediumFont = 15;
+
+  final Movie movie = Movie.defaultMovie();
+
+  @override
+  Widget build(BuildContext context) {
+    LikeButton likeButton =  LikeButton(movie: movie,);
```

Comment:
Why this instance creation here? It's better to create the instance directly in the widget tree below.

Reply:
fixed